git/guides/commit-message.md

```diff
@@ -0,0 +1,52 @@
+# 💬 Commit message convention
+
+## Recommendation
+
+1. Use [Gitmoji](https://gitmoji.carloscuesta.me/)
+    - Use single emoji, use unicode (🐛, not `:bug:`), seperate with single space from message
+    - Use only valid Gitmojis and respect the semantics to your best judgement and knowlidge (if multiple apply, select the more specific)
```

Comment:
Typo in `knowlidge`.

Reply:
Thanks :eagle: :eye: 

Checked all for typos and found another one. Amended.

```diff
 diff --git a/git/guides/commit-message.md b/git/guides/commit-message.md
index e48b021..c2dd3eb 100644
--- a/git/guides/commit-message.md
+++ b/git/guides/commit-message.md
@@ -4,10 +4,10 @@
 
 1. Use [Gitmoji](https://gitmoji.carloscuesta.me/)
     - Use single emoji, use unicode (🐛, not `:bug:`), seperate with single space from message
-    - Use only valid Gitmojis and respect the semantics to your best judgement and knowlidge (if multiple apply, select the more specific)
+    - Use only valid Gitmojis and respect the semantics to your best judgement and knowledge (if multiple apply, select the more specific)
 2. Write message titles according to [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
     - Briefly and precisely describe what/how/why, add detail to body if required
-    - Blank lines to seperate description/body/trailers, 50/72 lengths
+    - Blank lines to separate description/body/trailers, 50/72 lengths
     - Capitalize message, use imperative, omit trailing dot
 3. Reference RM issue in body as `#12345`
     - If you need to reference GH and RM, use `#GH-123` to avoid collision (does not break either integration)
```